src/api/controllers/bookmarkController.js

```diff
@@ -0,0 +1,68 @@
+import status from '../../helpers/constants/status.codes';
+import errorSender from '../../helpers/error.sender';
+import errorMessage from '../../helpers/constants/error.messages';
+import model from '../models/index';
+
+const { Bookmark } = model;
+/**
+ * @class
+ */
+export default class BookmarkController {
+  /**
+   *
+   * @param {*} req
+   * @param {*} res
+   * @return {object} void
+   */
+  static async addBookmark(req, res) {
+    const user = req.user.username;
+    const { slug } = req.params;
+
+    try {
+      await Bookmark.create({
+        slug,
+        username: user
+      });
+
+      res.status(status.CREATED).json({
+        status: 201,
+        message: 'Bookmarked successfully'
+      });
+    } catch (e) {
+      await Bookmark.destroy({
+        where: {
+          username: user,
+          slug
+        }
+      });
+      res.status(status.OK).json({
+        status: 200,
+        message: 'Successfully removed bookmark'
+      });
+    }
+  }
+
+  /**
+ *
+ * @param {*} req
+ * @param {*} res
+ * @return {object} void
+ */
+  static async getBookmark(req, res) {
+    const user = req.user.username;
+    const emptyBookmark = 0;
+
+    const bookmarkList = await Bookmark.findAll({
+      where: {
+        username: user,
+      }
+    });
+    if (bookmarkList.length > emptyBookmark) {
+      res.status(status.OK).json({
```

Comment:
please, add the status in the body @truestbyheart

Reply:
Thanks, @nkpremices +1 for you